### ISSUE TITLE
Breaking changes after 5.2.4 -> 5.3.0 update

### ISSUE THREAD

**Issue by krtr** [pixijs/pixi.js#6728]
I was unable to find info if pixi is sticking to semantic versioning or not.
Anyway if it is not please feel free to close this issue

### Expected Behavior

Minor version bump does not introduce breaking changes

### Current Behavior

Minor version bump introduced breaking changes

### Possible Solution

Don't introduce breaking change with minor version bump 

### Steps to Reproduce

Examples:

#### 1. 

https://github.com/pixijs/pixi.js/blob/v5.2.4/types/loader.d.ts#L28 contains `on`/`once`/`off` methods
https://github.com/pixijs/pixi.js/blob/v5.3.0/types/loader.d.ts does not contain `on`/`once`/`off` methods

#### 2.

https://github.com/pixijs/pixi.js/blob/v5.2.4/packages/core/src/Renderer.js#L63 has `forceFXAA` prop
https://github.com/pixijs/pixi.js/blob/v5.3.0/packages/core/src/Renderer.ts does not have `forceFXAA`

#### 3.

not easly trackable error with BitmapText:

```
TS2345: Argument of type 'TPIXIBitmapTextConstructorStyle' is not assignable to parameter of type '{ fontName: string; fontSize?: number | undefined; align?: string | undefined; tint?: number | undefined; letterSpacing?: number | undefined; maxWidth?: number | undefined; }'.
```

I can create repo with code that will compile under 5.2.4 but will break with 5.3.0 if that helps

btw  first two errors seems to be easy fix, IDK about 3

### Environment

- **`pixi.js` version**: 5.2.4 -> 5.3.0
- **Browser & Version**: Does not apply
- **OS & Version**: Does not apply
- **Running Example**: If deemed necessary

**Comment by bigtimebuddy** [comment]
1. This is intentional. We never properly deprecated events on Loader and it was challenging to convert to TypeScript because of the EventEmitter mixin. Any code that was previous build using events should continue to work runtime, but rebuilding you'll need to update to use the signals instead of events. I'll add a note about this on the release, thanks for pointing out.

2. The option `forceFXAA` was never used. This was a hold over from v3 or v4 that never got removed. Please discard if you were using this in your code.

3. We need more information on this. Can you just provide a reproduces just this issue?

Basically, we are in the process of upgrading our typings to be more specific. In the future we should consider type changes more carefully. In general, we've only been concerned about breaking _runtime_ changes, not _build-time_ changes. I imagine this will start to change in our next major release.
